packages/service-integration/src/service_integration/commands/config.py

```diff
@@ -88,11 +88,19 @@ def main(
     """Creates osparc config from complete docker compose-spec"""
     # TODO: sync defaults among CLI commands
     config_dir = from_spec_file.parent / ".osparc"
+    project_cfg_path = config_dir / "docker-compose.overwrite.yml"
     meta_cfg_path = config_dir / "metadata.yml"
     runtime_cfg_path = config_dir / "runtime.yml"
 
     meta_cfg_path.parent.mkdir(parents=True, exist_ok=True)
     runtime_cfg_path.parent.mkdir(parents=True, exist_ok=True)
     rich.print(f"Creating {config_dir} from {from_spec_file} ...")
 
-    create_osparc_specs(from_spec_file, meta_cfg_path, runtime_cfg_path)
+    create_osparc_specs(
+        from_spec_file, project_cfg_path, meta_cfg_path, runtime_cfg_path
+    )
+
+
+if __name__ == "__main__":
+    # pylint: disable=no-value-for-parameter
+    main()
```

Comment:
I guess you want this here.

Reply:
i will remove those in the next PR ... it was to test locally